Gopkg.toml

```diff
@@ -4,10 +4,10 @@ ignored = ["github.com/elastic/beats*"]
   name = "github.com/aws/aws-sdk-go"
   version = "1.16.14"
 
-[prune]
-  go-tests = true
-  unused-packages = true
-
 [[constraint]]
   name = "github.com/rs/xid"
   version = "1.1.0"
+
+[[constraint]]
```

Comment:
These are go requirements?

Reply:
Yes. We import this backoff library to do exponential backoff.